src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,317 @@
+            ((reason) => {
+              console.error(reason);
+            })
+        );
+    }
+  }, []);
+
+  type iconMapObject = Record<string, JSX.Element>;
+
+  const iconMap: iconMapObject = {
+    linkedin: <LinkedIcon />,
+    twitter: <TwitterIcon />,
+    instagram: <InstagramIcon />,
+  };
+
+  return (
+    <Box
+      sx={{
+        borderTop: 1,
+        borderColor: 'divider',
+        backgroundColor: 'background.paper',
+      }}
+    >
+      <Container
+        maxWidth={false}
+        sx={{ py: 8, display: 'flex', justifyContent: 'center' }}
+      >
+        <Grid
+          container
+          spacing={{ xs: 6, sm: 3 }}
+          xs={8}
+          justifyContent="center"
+        >
+          <Grid item xs={12} sm={3}>
+            <Stack spacing={2} alignItems={{ xs: 'center', sm: 'start' }}>
+              {companyLink && (
+                <Link
+                  component="button"
+                  aria-label={companyLink.ariaLabel}
+                  href={companyLink.href ?? hrefNoOp}
+                  onClick={wrapHandleExitAction(
+                    companyLink.href ?? hrefNoOp,
+                    companyLink.onClick,
+                    onExit
+                  )}
+                  sx={{ display: 'inline-flex' }}
+                >
+                  <LogoPagoPACompany />
+                </Link>
+              )}
+
+              <Stack
+                component="ul"
+                alignItems={{ xs: 'center', sm: 'start' }}
+                textAlign="center"
+                sx={{ padding: 0, listStyle: 'none' }}
+              >
+                {aboutUs?.links.map(
+                  ({ href = hrefNoOp, label, ariaLabel, onClick }, i) => (
+                    <li key={i}>
+                      <Link
+                        aria-label={ariaLabel}
+                        component="a"
+                        href={href}
+                        onClick={wrapHandleExitAction(href, onClick, onExit)}
+                        underline="none"
+                        color="text.primary"
+                        sx={{ display: 'inline-block', py: 0.5 }}
+                        variant="subtitle2"
+                      >
+                        {label}
+                      </Link>
+                    </li>
+                  )
+                )}
+              </Stack>
+            </Stack>
+          </Grid>
+          {!hideProductsColumn && (
+            <Grid item xs={12} sm={3}>
+              <Stack spacing={2} alignItems={{ xs: 'center', sm: 'start' }}>
+                {jsonProducts && (
+                  <Typography variant="overline">Prodotti e Servizi</Typography>
+                )}
+
+                <Stack
+                  component="ul"
+                  alignItems={{ xs: 'center', sm: 'start' }}
+                  textAlign="center"
+                  sx={{ padding: 0, listStyle: 'none' }}
+                >
+                  {jsonProducts &&
```

Comment:
You can remove the jsonProducts &&, leaving only jsonProducts?.map and remove the disable rule on top of the file.

Reply:
do it